testing/bdd/features/validation/sequence_id_num.feature

```diff
@@ -1,6 +1,11 @@
 @validation @syntax @sequence @xsd
 Feature: Sequence ID and Sequence Number
-  Both are mandatory parameters and the sequence number has to be a number (no letters)
+  Both are mandatory parameters and the sequence number has to be a number (no letters).
+  Moreover two documents with the same sequence identifier shall have different sequence numbers.
```

Comment:
The only way I see to apply the comparison to everything is to compare the xml produced by the bindings directly. @kozmaz87 do you see any other way ?